config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -154,6 +161,8 @@ presubmits:
           env:
             - name: NODE_FEATURE_GATES
               value: "PodAndContainerStatsFromCRI=true"
```

Comment:
should we add the HPA metrics feature gate too?

Reply:
the default tests we run don't include it since we have those covered via `pull-kubernetes-e2e-capz-windows-serial-slow-hpa`